src/main/scala/cromwell/engine/workflow/WorkflowOptions.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.engine.workflow
+
+import com.typesafe.config.ConfigFactory
+import cromwell.util.{EncryptedBytes, SecretKey, Aes256Cbc}
+import scala.collection.JavaConverters._
+import spray.json._
+import scala.util.{Try, Success, Failure}
+
+object WorkflowOptions {
+  private lazy val WorkflowOptionsConf = ConfigFactory.load.getConfig("workflow-options")
+  private lazy val EncryptedFields: Seq[String] =
+    WorkflowOptionsConf.getList("encrypted-fields").unwrapped().asScala.toSeq.asInstanceOf[Seq[String]]
```

Comment:
Also, if you must cast there's no need to use the type ascription - it's pretty obvious what it is